Documentation/Prerequisites/Index.rst

```diff
@@ -0,0 +1,39 @@
+.. include:: /Includes.rst.txt
+
+.. _prerequisites:
+
+=============
+Prerequisites
+=============
+
+Before you start working on this tutorial you should have a working TYPO3
+installation in versions 13.3 or above.
+
+We assume you use Composer and work locally with DDEV.
+
+Your TYPO3 installation should be empty and have no pages, designs or third
+party extensions installed.
+
+You can follow this tutorial to install TYPO3 with DDEV and Composer:
+
+*   :ref:`Installing TYPO3 with DDEV <t3start:installation-ddev-tutorial>`.
+
+..  note:: Answer "no" when asked if you want to create a basic site.
+
+You should be able to look into the TYPO3 backend and see the following:
+
+..  figure:: EmptyInstallation.png
+    :alt: Screenshot of a fresh TYPO3 13.4-dev installation with no page in the page tree
+
+    A fresh, empty TYPO3 installation (light mode)
+
+It is also helpful if you know the following concepts:
+
+*   Backend
```

Comment:
Can we link at least some of them for further reading?

Reply:
I created an issue for that as most of them have not yet properly been explained in "Getting Started": https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage/issues/188